Execution Result for prediction_script.py
--------------------------------------------------
Execution error (not missing module). Attempts: 1
LOG_START:WORKING_DIR
Working in: /workspace/pipeline_e3e0f8c3-d0c6-4620-9df5-fdf5ac58d4e8
LOG_END:WORKING_DIR
Checking whether there is an H2O instance running at http://localhost:54321..... not found.
Attempting to start a local H2O server...
  Java Version: openjdk version "21.0.8" 2025-07-15; OpenJDK Runtime Environment (build 21.0.8+9-Debian-1); OpenJDK 64-Bit Server VM (build 21.0.8+9-Debian-1, mixed mode, sharing)
  Starting server from /usr/local/lib/python3.11/site-packages/h2o/backend/bin/h2o.jar
  Ice root: /tmp/tmpsxsf4ocr
  JVM stdout: /tmp/tmpsxsf4ocr/h2o_unknownUser_started_from_python.out
  JVM stderr: /tmp/tmpsxsf4ocr/h2o_unknownUser_started_from_python.err
  Server is running at http://127.0.0.1:54321
Connecting to H2O server at http://127.0.0.1:54321 ... successful.
Warning: Your H2O cluster version is (5 months and 8 days) old.  There may be a newer version available.
Please download and install the latest version from: https://h2o-release.s3.amazonaws.com/h2o/latest_stable.html
--------------------------  ----------------------------------
H2O_cluster_uptime:         01 secs
H2O_cluster_timezone:       Etc/UTC
H2O_data_parsing_timezone:  UTC
H2O_cluster_version:        3.46.0.7
H2O_cluster_version_age:    5 months and 8 days
H2O_cluster_name:           H2O_from_python_unknownUser_wy29nq
H2O_cluster_total_nodes:    1
H2O_cluster_free_memory:    1.983 Gb
H2O_cluster_total_cores:    16
H2O_cluster_allowed_cores:  16
H2O_cluster_status:         locked, healthy
H2O_connection_url:         http://127.0.0.1:54321
H2O_connection_proxy:       {"http": null, "https": null}
H2O_internal_security:      False
Python_version:             3.11.13 final
--------------------------  ----------------------------------
LOG_START:H2O_INIT
H2O initialized successfully
LOG_END:H2O_INIT
LOG_START:MODEL_LOADED
Model loaded: /workspace/pipeline_e3e0f8c3-d0c6-4620-9df5-fdf5ac58d4e8/DeepLearning_grid_1_AutoML_1_20250904_184011_model_7
Model type: H2ODeepLearningEstimator
LOG_END:MODEL_LOADED
ERROR_START:'response_column':ERROR_END
H2O session _sid_906c closed.

